app/src/androidTest/java/com/ncnf/authentication/LoginActivityTest.java

```diff
@@ -100,7 +115,7 @@ public void signInFragmentUnsuccessfulLoginTest(){
 
         verify(mockedAuth).logIn(anyString(), anyString());
 
-        onView(withId(R.id.exceptionSignIn)).check(matches(withText(containsString(unsuccessfulLogin))));
+        onView(withId(android.R.id.button1)).check(matches(isClickable()));
```

Comment:
The id button1 is not meaningful.

Reply:
It's the default id for the button of the pop-up I do not think I can change it

Reply:
Fine then, my bad